contracts/nois-proxy/src/contract.rs

```diff
@@ -78,10 +87,20 @@ pub fn instantiate(
         .add_attribute("test_mode", test_mode.to_string()))
 }
 
-// This no-op migrate implementation allows us to upgrade within the 0.7 series.
-// No state changes expected.
 #[cfg_attr(not(feature = "library"), entry_point)]
-pub fn migrate(_deps: DepsMut, _env: Env, _msg: Empty) -> StdResult<Response> {
+pub fn migrate(deps: DepsMut, env: Env, _msg: Empty) -> StdResult<Response> {
+    let mut config = CONFIG.load(deps.storage)?;
+
+    // If unset, set min_after and max_after to now and now+10y
+    if config.min_after.is_none() {
```

Comment:
Maybe I agree with you now when you said few months ago that we can just not  allow people to request past rounds. Then we simply remove this min_after and simply calculate it during the request by assigning the current block time.

Reply:
That is what I tried first. The problem with this is that current block time is always the local time of the machine in the CI. Then all integration tests fail because they use some old drand round in the testing data.

Reply:
Alright, then we'd need to update this [table](https://docs.nois.network/integrate_nois_to_your_chain.html#instantiate-the-nois-proxy-contract-on-your-chain) to make sure it's clear enough for people